hello-word/src/components/Jokes.js

```diff
@@ -0,0 +1,20 @@
+import React, { useState,useEffect } from 'react';
+import Data from "./JokesData";
+
+export default function Jokes() {
+    const [jokes, setJokes] = useState([]);
+    useEffect(() => {
+        setJokes(Data); 
+    }, []);
+
+    return (
+        <div>
+            {jokes.map((joke, index) => (
+                <div className='joke-name' key={index}>
```

Comment:
When using the array index as the 'key' is common, it can lead to problems if the list changes. It's best to use a unique identifier as the 'key' to help React work smoothly. Try using something like an ID for better results.